DESCRIPTION

```diff
@@ -78,16 +64,16 @@ Suggests:
     waldo,
     xml2
 LinkingTo: 
-    progress (>= 1.2.1),
     cpp11 (>= 0.2.0),
+    progress (>= 1.2.1),
     tzdb (>= 0.1.1)
 VignetteBuilder: 
     knitr
-Config/testthat/edition: 3
-Config/testthat/parallel: false
 Config/Needs/website:
-    nycflights13,
+    nycflights13, 
```

Comment:
This is very much a nitpick. But there should be no trailing whitespace here. It's a good nudge to configure any text editor / IDE you're using to remove trailing whitespace in files treated as source code (but you would not want this for, say, markdown). Good default config will pay off in the long run.

In RStudio, do *Tools > Global Options > Code > Saving* and I recommend:
* YES Ensure that source files end with newline
* YES Strip trailing horizontal whitespace when saving <-- SPECIFIC THING WE'RE TALKING ABOUT RIGHT NOW